uiowa/uiowa
# SROP scholar image Display

Similar to #2939, the images are showing on the SROP scholar listing pages as an oval.

# How to test


1. Checkout this branch
2. sync down grad.uiowa.edu and make sure all config is imported
3. observe https://grad.local.drupal.uiowa.edu/dei/srop/past-scholars/2019 and see that the scholar images on the left are now circles.
